routing/payment_session.go

```diff
@@ -28,15 +27,15 @@ type PaymentSession interface {
 	// channel failed to route the previous payment attempt. The
 	// PaymentSession will use this information to produce a better next
 	// route.
-	ReportEdgeFailure(e *EdgeLocator)
+	ReportEdgeFailure(failedEdge edge, minPenalizeAmt lnwire.MilliSatoshi)
 
 	// ReportEdgePolicyFailure reports to the PaymentSession that we
 	// received a failure message that relates to a channel policy. For
 	// these types of failures, the PaymentSession can decide whether to to
 	// keep the edge included in the next attempted route. The
 	// PaymentSession will use this information to produce a better next
 	// route.
-	ReportEdgePolicyFailure(errSource route.Vertex, failedEdge *EdgeLocator)
+	ReportEdgePolicyFailure(failedEdge edge)
```

Comment:
Does this change penalization from node to "node pair"?

Reply:
No there is no change here, just a different struct that contains both parameters.